inspector/src/components/persistentPopupHostComponent.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { PopupComponent } from "./popupComponent";
+import { Nullable } from "babylonjs/types";
+import { GlobalState } from "./globalState";
+import { Observer } from "babylonjs/Misc/observable";
+
+interface IPersistentPopupHostComponentProps {
+    globalState?: GlobalState;
+}
+
+interface IPersistentPopupHostComponentState {
+    renderFunction?: Nullable<() => React.ReactNode>;

```

Comment:
Why is this optional? Is there a case where we open a pop up and don't require to render anything there?

Reply:
For the wrapper, if it there isn't a render function it doesn't render the popup, but this could be just nullable, no need for the optional.